app.py

```diff
@@ -733,6 +734,20 @@ def prepare_model_args(request_body):
         if message:
             messages.append({"role": message["role"], "content": message["content"]})
 
+    user_json = None
+    if (MS_DEFENDER_ENABLED):
+        authenticated_user_details = get_authenticated_user_details(request_headers)
+        tenantId = get_tenantid(authenticated_user_details.get("client_principal_b64"))
+        conversation_id = request_body.get("conversation_id", None)        
+        user_args = {
```

Comment:
`user_args` is what my previous comment was referring to.  I think I just forgot to click here to add that comment.  :)

Reply:
Right! 
I did test it with None values without any issue